### ISSUE TITLE
Make create group button Accessible

### ISSUE THREAD

**Issue by HoldUpFjord** [LetsGetTechnical/elecretanta#132]
## Feedback from Hackathon
The shade of yellow for the Create group button is likely not accessible, how do we want to approach this? 

![Image](https://github.com/user-attachments/assets/80a2701e-01b2-46e6-a264-525aac158be6)

**Comment by Danielle254** [comment]
@allyroo This button is also used for the Google Login on the landing page. Do you have a suggestion for the new design color? I can provide the current accessibility score and a suggested new "yellow" color if that would help!

**Comment by Danielle254** [comment]
WebAIM color contrast accessibility checker confirms this combination fails WCAG standards

![Image](https://github.com/user-attachments/assets/86e1f7da-2a41-4c08-80f7-91f73414dfbd)
